src/main/scala/com/ironcorelabs/GroupAccessEditResult.scala

```diff
@@ -0,0 +1,16 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+
+case class GroupAccessEditResult(
+  succeeded: List[UserId],
+  failed: List[GroupAccessEditErr]
+)
+
+object GroupAccessEditResult {
+  def apply(gaer: jsdk.GroupAccessEditResult): GroupAccessEditResult =
+    GroupAccessEditResult(
+      gaer.getSucceeded.map(id => UserId(id.getId)).toList,
+      gaer.getFailed.map(err => GroupAccessEditErr(UserId(err.getUser.toString), err.getError)).toList
```

Comment:
You shouldn't have to `toString` this. Also, I don't think `toString` would have been the right call to make `getId` is. Either way, we have a UserId apply function that takes the java type.

Reply:
Good catch, I definitely intended to do `.getId`. But I've now removed that and just used the java type constructor.